### PR TITLE
SJ - SRY - Modify past statuses

### PR DESCRIPTION
**Pull request by Stuart-Johnson**
This has three steps.

A: Create "new status" column on past_statuses, and populate
B: Migrate existing "obtain research pricing" ssrs and past_statuses over to "get a cost estimate"
C: Rake task to fill in the various kinds of messed up past_status data, as well as missing submitted_at dates on ssrs.

Three stories:
[#163157889] [#163269655] [#163337270]

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,74 +10,74 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_11_28_192700) do
+ActiveRecord::Schema.define(version: 2019_01_18_194740) do
 
-  create_table "admin_rates", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci", force: :cascade do |t|
-    t.bigint "line_item_id"
+  create_table "admin_rates", id: :integer, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci", force: :cascade do |t|
```
Comment (kyle-glick) [inline db/schema.rb]: These changes shouldn't be included as we normally have been using BIGINTs, not INTEGERs for our ID columns for a while. I don't recall if that was done by a migration or rake task but you probably want to run that locally and remove those schema changes

File: lib/tasks/fix_past_status_data.rake [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# Copyright © 2011-2018 MUSC Foundation for Research Development~
```
Comment (kyle-glick) [inline lib/tasks/fix_past_status_data.rake]: One more thing I noticed. Can you update the copyright for this file to 2019

File: db/schema.rb (hunk 1 above)
Comment (Stuart-Johnson) [reply db/schema.rb]: Yeah, not sure how this got in, removed.

File: lib/tasks/fix_past_status_data.rake (hunk 2 above)
Comment (Stuart-Johnson) [reply lib/tasks/fix_past_status_data.rake]: Yup, done

**Pull request merged**